internal/server/v2/event/golden/event_collection/DroughtEvent_BFA_202205.json

```diff
@@ -1,87 +1,3 @@
 {
-  "eventCollection": {
-    "events": [
-      {
-        "places": [
-          "Earth",
-          "africa",
-          "country/BFA",
-          "country/MLI",
-          "country/NER",
-          "grid_1/12_2",
-          "grid_1/13_0",
-          "grid_1/13_1",
-          "grid_1/13_2",
-          "grid_1/13_3",
-          "grid_1/14_1",
-          "grid_1/14_2",
-          "grid_1/14_3",
-          "grid_1/14_4",
-          "grid_1/15_1",
-          "grid_1/15_2",
-          "grid_1/15_3",
-          "grid_1/15_4",
-          "grid_1/16_0",
-          "grid_1/16_1",
-          "grid_1/16_2",
-          "grid_1/16_3",
-          "grid_1/16_4",
-          "grid_1/17_0",
-          "grid_1/17_1"
-        ],
-        "geoLocations": [
-          {
-            "point": {
-              "latitude": 12,
-              "longitude": 2
-            }
-          }
-        ],
-        "dates": [
-          "2022-05"
-        ],
-        "propVals": {
-          "area": {
-            "vals": [
-              "SquareKilometer250168.38146"
-            ]
-          },
-          "endDate": {
-            "vals": [
-              "2022-06"
-            ]
-          },
-          "geoJsonCoordinatesDP1": {
-            "vals": [
-              "{\"type\": \"Polygon\", \"coordinates\": [[[3.5, 13.5], [3.5, 12.5], [2.5, 12.5], [2.5, 11.5], [1.5, 11.5], [1.5, 12.5], [-0.5, 12.5], [-0.5, 13.5], [0.5, 13.5], [0.5, 15.5], [-0.5, 15.5], [-0.5, 17.5], [1.5, 17.5], [1.5, 16.5], [4.5, 16.5], [4.5, 13.5], [3.5, 13.5]]]}"
-            ]
-          },
-          "name": {
-            "vals": [
-              "DroughtEvent at LatLong(12.00000:2.00000) on 2022-05"
-            ]
-          },
-          "observationPeriod": {
-            "vals": [
-              "P32D"
-            ]
-          },
-          "standardizedPrecipitationIndex": {
-            "vals": [
-              "-3.09"
-            ]
-          }
-        },
-        "dcid": "droughtEvent/2022-05_grid_1/12_2",
-        "provenanceId": "dc/base/NOAA_GPCC_StandardardizedPrecipitationIndex_Event"
-      }
-    ],
-    "provenanceInfo": {
-      "dc/base/NOAA_GPCC_StandardardizedPrecipitationIndex_Event": {
-        "importName": "NOAA_GPCC_StandardardizedPrecipitationIndex_Event",
-        "provenanceUrl": "https://www.drought.gov/data-maps-tools/global-precipitation-climatology-centre-gpcc-standardized-precipitation-index-spi",
-        "domain": "noaa.gov"
-      }
-    }
-  }
+  "eventCollection": {}
```

Comment:
This dataset was refreshed, it could be that this event went away.

@ajaits to confirm.

Reply:
There is an event in country/BFA with a different id:
droughtEvent/2022-12-01_grid_1/-10_-58
The dcid is not stable as it depends on the order of data in the source.

Reply:
TODO: could you please update the golden with a valid event?